src/lib/factories/leads.ts

```diff
@@ -55,6 +56,27 @@ const searchCallLeadDefaults: SearchCallLeadType = {
   duration: 0,
 }
 
+const searchWhatsappLeadDefaults: SearchWhatsappLeadType = {
+  createdDate: "2021-01-01T12:34:56.000Z",
+  firstName: "Birra",
+  id: 501,
+  lastName: "Peroni",
+  listing: {
+    externalListingId: "1234567",
+    image:
+      "2019/09/17/11/13/13/1-aixam-mac-500-cabriolet-480-261485-7e5hF5fhFUiC.jpg",
+    id: 555,
+    make: "Ford",
+    mileage: 6000,
+    model: "Fiesta",
+    price: 20000,
+    referenceId: "123",
+    type: "1.0 EcoB 125 Titanium X",
+  },
```

Comment:
Observation - we repeat that for every lead factory. Would it make sense to extract that to a factory of its own?